src/styles/components/site-footer.styl

```diff
@@ -0,0 +1,8 @@
+.site-footer
+  flex 0 0 auto  //Layout: Sticky footer. Sticks to bottom of window or bottom of page, whichever is larger.
+  
+  //The following isn't important and can be discarded.
```

Comment:
Ah, the "can be discarded" bits are just there to style the placeholder footer; they can be replaced with anything or removed entirely (although the footer would look ugly as heck). I only made that note because the only really important bit of CSS, layout-wise, is `flex: 0 0 auto`; every other bit of CSS is just deco so we can see _a_ footer.